netboot/netconf.go

```diff
@@ -1,16 +1,18 @@
 package netboot
 
 import (
+	"bytes"
+	"encoding/binary"
 	"errors"
 	"fmt"
+	"github.com/insomniacslk/dhcp/dhcpv4"
```

Comment:
I think we want to keep the github imports in a different group:
https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Got it, will amend.